### PR TITLE
new content mgmt screen and edit to secret overview

### PR DESCRIPTION
**Pull request by marcfishelquantil**
I updated two content management screens and made minor edits to the overview page for secrets.

### CODE REVIEW

File: docs/portal/secrets/creating-secrets.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ To create a secret, complete the Create a Secret form with required information.
 
 | **Fields**                | **Description**                                    |
 | --------------------------| ---------------------------------------------------|
-| Secret Name             | Enter a unique name that helps you identify this secret. The name is case sensitive and can be 30 characters long. You can include spaces between non-space characters; however, if you add spaces at the beginning or end of the secret, the spaces will be removed. The secret name must not contain special characters such as ^ " ~ , . # $ %|
+| Secret Name             | Enter a unique name that helps you identify this secret. The name is case sensitive and can be 30 characters long. You can include spaces between non-space characters; however, if you add spaces at the beginning or end of the secret, the space will be removed. The secret name must not contain special characters such as ^ " ~ , . # $ %|
```
Comment (burtonquantil) [inline docs/portal/secrets/creating-secrets.md:18]: Why was this changed to singular?

**Review by burtonquantil — APPROVED** [review]
OK to merge, but please check the comment I made.

File: docs/portal/secrets/creating-secrets.md (hunk 1 above)
Comment (marcfishelquantil) [reply docs/portal/secrets/creating-secrets.md:18]: Will fix and re-upload.

File: docs/portal/content-management.md [hunk 2]
```diff
@@ -12,7 +12,8 @@ The following figure shows the key elements on the page, and the table following
 | :----------: | --------------- |
 | 1 | To filter purges, type characters in this field and then press the Enter key. All purges that do not contain the typed characters are hidden. Filtering is not case-sensitive. To remove the filter, click the **x** icon at the right side of the search field. |
 | 2 | Each purge appears on its own row. Clicking an ID shows detailed information about the purge.|
-| 3 | The button to create purge requests.|
+| 3 | Icon to filter purges by Completed date and time.|
```
Comment (burtonquantil) [inline docs/portal/content-management.md:15]: Upon further review, this is actually incorrect.  The icon allows the user to filter purges by their submission date.
The icon is next to the Submitted column.

**Review by burtonquantil — CHANGES_REQUESTED** [review]
Please check the comment. Look at the portal.

**Review by burtonquantil — APPROVED** [review]
OK to merge.
